docs/UserGuide.md

```diff
@@ -78,13 +73,97 @@ Fig 1. Home tab
 {:.caption}
 
 #### 2.2.2 Inventory & Statistics Tab
+
 Displays [inventory](#32-inventory) and [statistics](#35-statistics).
 
 ![Inventory & Statistics tab](images/product-screenshots/general/BarChart.png)
 
 Fig 2. Inventory & Statistics tab
 {:.caption}
 
+### 2.3 Creating your first order
+
+This is a step-by-step guide to creating your first order. If you have not installed BobTheBistroBoss, you
+may do so by following the instructions [here](#21-installation).
+
+For this guide, we will simulate the following scenario:
+
+John Doe wishes to place two orders, one on 10 October 2021 and the other on 11 October 2021. Both of these orders are
+to be delivered at 12pm. For both of these orders, he will require 3 tuna sandwiches. Each tuna sandwich will be priced
+at $2.
+
+Here are the details of John Doe:
+* Name: John Doe
+* Phone Number: 92345678
+* Email: johndoe@gmail.com
+* Address: Blk 422, Bedok North Road, #15-25
+
+Here are the ingredients that will be used for the tuna sandwich:
+
+* 2 slices of bread
+* 1 slice of cheese
+* 1 can of tuna
+
+Here are the quantities of the respective ingredients in our inventory:
+
+* 100 slices of bread
+* 100 slices of cheese
+* 100 cans of tuna
+
+#### 2.3.1 Approach 1
+
+We can proceed to add the first order on 10 October 2021 by entering the following
+command in the command box at the bottom of the app:
+
+`add-o cn/John Doe cp/92345678 ca/Blk 422, Bedok North Road, #15-25 rn/Tuna Sandwich ri/Bread-2-slices,Cheese-1-slices,Tuna-1-cans op/6 oq/3 od/10-10-2021 1200`
+
+You will notice that the order has been added at the 4th index in the Orders list. However, this is not the
```

Comment:
These indexes are hardcoded. I think it is best to remove this so that we dont have to worry about changing this in the future if we were to remove some of the sample data stuff